base_layer/core/src/mempool/orphan_pool/orphan_pool_storage.rs

```diff
@@ -46,14 +45,8 @@ impl<T> OrphanPoolStorage<T>
 where T: BlockchainBackend
 {
     /// Create a new OrphanPoolStorage with the specified configuration
-    pub fn new(
-        blockchain_db: BlockchainDatabase<T>,
-        config: OrphanPoolConfig,
-        validator: Validator<Transaction, T>,
-    ) -> Self
-    {
+    pub fn new(config: OrphanPoolConfig, validator: Validator<Transaction, T>) -> Self {
         Self {
-            blockchain_db,
```

Comment:
OMG. Testing must be so much easier now

Reply:
Yes, I think it will make the tests much easier.